packages/lit-virtualizer/README.md

```diff
@@ -163,20 +185,43 @@ render(templateResult, document.querySelector("#post-list"));
 
 With `<lit-virtualizer>`, you pass configuration as properties to the HTML Element.
 
-### `<lit-virtualizer>` properties
+### `<lit-virtualizer>` API
+
+#### `items` property
+
+Type: `Array<T>`
+
+A list of items to display via the template function. The type of the items should match the first argument of the template function.
+
+#### `template` property
 
-`.items = Array<T>`
+Type: `(item: T, index?: number) => lit-html.TemplateResult`
 
-> A list of items to display via the template function. The type of the items should match the first argument of the template function.
+A function that returns a lit-html TemplateResult. It will be used to generate the DOM for each item in the virtual list.
 
-`.template = (item: T, index?: number) => lit-html.TemplateResult`
+#### `scrollTarget` property
 
-> A function that returns a lit-html TemplateResult. It will be used to generate the DOM for each item in the virtual list.
+Type: `Element | Window`
 
-`.scrollTarget = Element | Window`
+Optional. An element that receives scroll events for the virtual scroller. If not specified, the directive's parent element will be the target.
```

Comment:
If not specified, the `<lit-virtualizer>` element itself will be the scroll target.